src/multi_physics/QED_table_generator/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ set(
     ${CMAKE_BINARY_DIR}/bin
     )
 
-include_directories(../QED)
+include_directories(../.)
```

Comment:
The `.` looks confusing and should be superfluous.

Reply:
Here I wanted to be able to compile QED table generator separately (e.g. being able to do `mkdir build ; cd build ; cmake ../.` directly from `QED_table_generator` directory. If doing this is bad for some reason we can change it.

Reply:
I think we forgot to come back to this yesterday.

I would say create an `PXRMP::PXRMP` interface target in the `src/multi_physics/CMakeLists.txt` file and only give it `target_include_directories(PXRMP INTERFACE .)`.

You should then be able to express the QED table generator executable in terms of `target_link_libraries(qed_table_generator PRIVATE PXRMP::PXRMP)`

Reply:
Thanks for the advice! I should have followed it: I moved `QED_table_generator` into `QED` and I have added the option `PXRMP_QED_TOOLS` .

Reply:
However, Ubuntu tests are failing with this weird message:
```
Err:20 https://packagecloud.io/github/git-lfs/ubuntu focal InRelease
  402  Payment Required [IP: 54.183.38.243 443]
```